common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -104,7 +104,7 @@ public enum Feature {
 
         MULTI_SITE("Multi-site support", Type.DISABLED_BY_DEFAULT),
 
-        REMOTE_CACHE("Remote caches support. Requires Multi-site support to be enabled as well.", Type.EXPERIMENTAL),
+        STATELESS("External Infinispan support. Requires Multi-site support to be enabled as well.", Type.EXPERIMENTAL),
```

Comment:
```suggestion
        CLUSTERLESS("Store all session data, work cache and login failure data in an external Infinispan.", Type.EXPERIMENTAL),
```

Reply:
@pruivo JFYI - if the `MULTI_SITE` feature is required to be enabled as well, you can leverage the feature dependencies vararg for the feature (AFAIK). And you can omit the additional message in the description (or go with Alex's version).

```
STATELESS(".....", Type.EXPERIMENTAL, Feature.MULTI_SITE)
```

Reply:
@mabartos thanks for the tip. I wasn't aware of that :facepalm: